### PR TITLE
eq-125 Thankyou Page

### PR DESCRIPTION
**Pull request by hamishtaplin**
## Screenshot

screen shot 2016-04-06 at 14 36 38
## Changes
- added `.confirmation` component.
- added content to thankyou page
- added some utility classes for typography
## Test
- `npm run compile`
- visit thankyou page
- confirm it looks like screenshot above

### CODE REVIEW

File: app/templates/thank-you.html [hunk 1]
```diff
@@ -4,6 +4,22 @@
 
 {% block survey %}
 
-  <p>Thank you for submitting the survey {% if data.submitted_at %} on {{data.submitted_at}} {% endif %}</p>
+  <div class="header u-mb-m u-mb-l@s">
+    <h1 class="header__title">Thank You</h1>
+    <div>Monthly Business Survey - Retail Sales Index</div>
+    <div class="u-lighter">Period: 4th January 2016 to 31st January 2016</div>
```
Comment (hamishtaplin) [inline app/templates/thank-you.html]: ~~Are we hardcoding these values for now?~~

Actually, these should be the dates submitted by the user and so need to be wired up.

Comment (hamishtaplin) [inline app/templates/thank-you.html]: Having talked about it, this isn't easily achieved. @alexmorris to review wether we need it.

Comment (warrenbailey) [inline app/templates/thank-you.html]: As discussed with Alex we'll just remove the line with the dates for MCI

**Comment by warrenbailey** [comment]
LGTM :+1:

**Pull request merged**